Automattic/jetpack
# Fix: wrong applied styles on Multiple checkbox in Contact From block

Fixes https://github.com/Automattic/jetpack/issues/31397

## Proposed changes:
This PR should fix wrongly applied styles. In the current state, we are applying `opacity` and other styles on all `span` elements, while we should only apply them on "(required)" line.

Note: Even though issue states about Tazza theme, issue persists on all themes.

### Other information:
This issue is a bit odd, as it seems that Styles used here are from `deprecated` Forms module, while the code used is from the new package. I don't think we should iterate on this within this Fix, but, it would be resolved when we fully remove old Forms module.
cc @jeherve 


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Download the branch locally and run `jetpack build plugins/jetpack`
* Work with your JT or other jetpack setup environment
* Create a new post and add Jetpack Form to it
* Add a Multiple Choice (Checkbox) field
* In block editor change the Option Text color
* Save and check out the Post and verify that the styles the same as in Edit mode (see original issue for context on how it was before)
<img width="1126" alt="Screenshot at Oct 26 14-10-16" src="https://github.com/Automattic/jetpack/assets/60262784/cc14e262-732c-406f-a7c8-8eb2b82422bc">
